sdv/cag/_utils.py

```diff
@@ -0,0 +1,18 @@
+from sdv.cag._errors import PatternNotMetError
+
+
+def _validate_table_and_column_names(table_name, columns, metadata):
```

Comment:
There is also a `_validate_table_and_column_names` in `sdv/cag/utils.py`. Do you think we could keep only one method?